package.json

```diff
@@ -10,12 +10,17 @@
     "@types/node": "^20.3.1",
     "@types/react": "^18.2.12",
     "@types/react-dom": "^18.2.5",
+    "lodash.findindex": "^4.6.0",
+    "lodash.isequal": "^4.5.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-scripts": "5.0.1",
     "typescript": "^5.1.3",
     "web-vitals": "^2.1.4"
   },
+  "overrides": {
```

Comment:
Ohh cool, this fixes a dependency issue that we currently have in the main branch. I was able to build successfully in this branch.